styles/main.css

```diff
@@ -59,14 +47,10 @@ select {
 .left article {
   color: #2d4b1e;
   background-color: white;
-  margin: 20px;
+  margin: 20px 0;
```

Comment:
Just an understanding question, the 20px applies to the top and 0 applies to all the other sides?

Reply:
Almost, when there are two values the first value applies to top and bottom, the second value applies to left and right.
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax

Reply:
Ahhh that makes sense. Thank you !